### ISSUE TITLE
I have a error when kbox start.

### ISSUE THREAD

**Issue by ngocleanh** [kalabox/kalabox#784]
<pre>
info: BOOT2DOCKER =>  Starting up. maxRetries=3, max=3
info: BOOT2DOCKER =>  Initializing boot2docker [1].
info: BOOT2DOCKER =>  Bringing boot2docker up [1].
info: BOOT2DOCKER =>  Boot2docker is up.
Kalabox engine has been activated.
info: SERVICES =>  Startup services need to be started.
info: DOCKER =>  Stopping container. 9dc0c5faefaff8019889b3b1f2a1094bfa3ff6f6f3d86e0bf22950516ccc5922
info: DOCKER =>  Container already stopped. 9dc0c5faefaff8019889b3b1f2a1094bfa3ff6f6f3d86e0bf22950516ccc5922
info: DOCKER =>  Stopping container. 985a3883b2f12c31c1d7400f7650782428f394f717a0863e97b4d2e206876b3d
info: DOCKER =>  Container already stopped. 985a3883b2f12c31c1d7400f7650782428f394f717a0863e97b4d2e206876b3d
info: DOCKER =>  Stopping container. d6b9c96f8bc3dc2b41318325f9ce105215873d3d21265172e4e0b4db41255149
info: DOCKER =>  Container already stopped. d6b9c96f8bc3dc2b41318325f9ce105215873d3d21265172e4e0b4db41255149
info: DOCKER =>  Stopping container. c2b2f81987034631ef0d48b43b465a0e78c606f08efea26211a0a72585fafa7d
info: DOCKER =>  Container already stopped. c2b2f81987034631ef0d48b43b465a0e78c606f08efea26211a0a72585fafa7d
info: DOCKER =>  Stopping container. 75e397563edf5426fcb5dd1e6d866a670f005c3e800bee73bf9d49ec086e5e13
info: DOCKER =>  Container already stopped. 75e397563edf5426fcb5dd1e6d866a670f005c3e800bee73bf9d49ec086e5e13
info: DOCKER =>  Starting container 9dc0c5faefaff8019889b3b1f2a1094bfa3ff6f6f3d86e0bf22950516ccc5922.
info: DOCKER =>  Container started. 9dc0c5faefaff8019889b3b1f2a1094bfa3ff6f6f3d86e0bf22950516ccc5922
info: DOCKER =>  Starting container 985a3883b2f12c31c1d7400f7650782428f394f717a0863e97b4d2e206876b3d.
info: DOCKER =>  Container started. 985a3883b2f12c31c1d7400f7650782428f394f717a0863e97b4d2e206876b3d
info: DOCKER =>  Starting container d6b9c96f8bc3dc2b41318325f9ce105215873d3d21265172e4e0b4db41255149.
info: DOCKER =>  Container started. d6b9c96f8bc3dc2b41318325f9ce105215873d3d21265172e4e0b4db41255149
info: DOCKER =>  Starting container c2b2f81987034631ef0d48b43b465a0e78c606f08efea26211a0a72585fafa7d.
info: DOCKER =>  Container started. c2b2f81987034631ef0d48b43b465a0e78c606f08efea26211a0a72585fafa7d
info: DOCKER =>  Starting container 75e397563edf5426fcb5dd1e6d866a670f005c3e800bee73bf9d49ec086e5e13.
info: DOCKER =>  Container started. 75e397563edf5426fcb5dd1e6d866a670f005c3e800bee73bf9d49ec086e5e13
info: DOCKER =>  Creating ad hoc container "kalabox/debian:stable". name=kalabox_temp-83873c30-d740-4de2-a783-15c68d9f3e16, Cmd=[bash], Tty=true, Image=kalabox/debian:stable, Env=[APPDOMAIN=longplay.kbox, KBOX_CODEDIR=code, KALABOX=true, APPNAME=longplay, SSH_KEY=pantheon.kalabox.id_rsa, GITUSER=Ngoc Le Anh, GITEMAIL=leanhngoclt@snowbot.net, PANTHEON_ACCOUNT=leanhngoclt@snowbot.net], VolumesFrom=[kb_longplay_data]
info: DOCKER =>  Ad hoc container created. 90eb6769381dae39cf65a9d29d52ad5af761e641fcf2cb26968a3bb10f622a76
info: DOCKER =>  Starting ad hoc container "90eb6769381dae39cf65a9d29d52ad5af761e641fcf2cb26968a3bb10f622a76". Binds=[/Users/apple/longplay:/src:rw]
info: DOCKER =>  Ad hoc container started. 90eb6769381dae39cf65a9d29d52ad5af761e641fcf2cb26968a3bb10f622a76
info: DOCKER =>  Removing ad hoc container. 90eb6769381dae39cf65a9d29d52ad5af761e641fcf2cb26968a3bb10f622a76
info: DOCKER =>  Starting container bd50a744e43b12236d434a84785de60e2ec37149c8eeb16b85d270a14c13c534. PublishAllPorts=true, Binds=[/Users/apple/longplay:/src:rw]
info: DOCKER =>  Container started. bd50a744e43b12236d434a84785de60e2ec37149c8eeb16b85d270a14c13c534
Error dispatching event listeners [post-start-component]: Error while running command "ln,-nsf,/media,/code/sites/default/files" in container bd50a744e43b12236d434a84785de60e2ec37149c8eeb16b85d270a14c13c534: HTTP code is 500 which indicates error: server error - Container bd50a744e43b12236d434a84785de60e2ec37149c8eeb16b85d270a14c13c534 is not running

VError: Error while running command "ln,-nsf,/media,/code/sites/default/files" in container bd50a744e43b12236d434a84785de60e2ec37149c8eeb16b85d270a14c13c534: HTTP code is 500 which indicates error: server error - Container bd50a744e43b12236d434a84785de60e2ec37149c8eeb16b85d270a14c13c534 is not running

    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:581:13
From previous event:
    at exec (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:580:11)
    at query (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:623:12)
    at Object.queryData (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:633:12)
From previous event:
    at Object.exports.queryData (/usr/local/bin/lib/engine.js.jx:649:27)
    at /Users/apple/longplay/plugins/kalabox-plugin-pantheon/lib/env.js:401:32
From previous event:
    at /Users/apple/longplay/plugins/kalabox-plugin-pantheon/lib/env.js:395:10
    at /usr/local/bin/lib/core/events.js.jx:149:11
    at /usr/local/bin/lib/core/events.js.jx:148:24
From previous event:
    at /usr/local/bin/lib/core/events.js.jx:132:23
From previous event:
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:129:18)
    at /usr/local/bin/lib/core/events.js.jx:108:19
From previous event:
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:100:21)
From previous event:
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:232:4)
    at /usr/local/bin/lib/app.js.jx:689:24
From previous event:
    at startComponent (/usr/local/bin/lib/app.js.jx:688:4)
    at services.verify.then.then.then.then.map.concurrency (/usr/local/bin/lib/app.js.jx:735:12)
From previous event:
    at Object.exports.start (/usr/local/bin/lib/app.js.jx:734:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/start.js.jx:22:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:857:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:173:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:236:12)
    at process._tickCallback (node.js:812:13)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:235:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:244:1)
    at Module._compile (module.js:518:26)
    at readX (module.js:1284:5)
    at Object.Module._extensions..jx (module.js:1711:10)
    at Module.load (module.js:345:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:377:17)
    at require (module.js:395:17)
    at Object.<anonymous> (_jx_source:17:3)
    at Module._compile (module.js:518:26)
    at Module.load (module.js:353:12)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:733:12)
    at startup (node.js:371:18)
    at node.js:1464:3
</pre>
Please tell me. Thank you.

**Comment by pirog** [comment]
@ngocleanh have you tried running start again? Are you on the latest version?

**Comment by rjbain** [comment]
I'm having this problem as well.

**Comment by rjbain** [comment]
I'm still having the same problem. It only happens with one of my sites, however.

I run `kbox start` and get this:

````
Error dispatching event listeners [post-start-component]: Error while running command "ln,-nsf,/media,/code/sites/default/files" in container 779e3c2340b5da52cdc43e8131b76236deb6eae9c509e81e25790b4b6163414f: HTTP code is 500 which indicates error: server error - Container 779e3c2340b5da52cdc43e8131b76236deb6eae9c509e81e25790b4b6163414f is not running

VError: Error while running command "ln,-nsf,/media,/code/sites/default/files" in container 779e3c2340b5da52cdc43e8131b76236deb6eae9c509e81e25790b4b6163414f: HTTP code is 500 which indicates error: server error - Container 779e3c2340b5da52cdc43e8131b76236deb6eae9c509e81e25790b4b6163414f is not running

    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:601:13
From previous event:
    at exec (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:600:11)
    at query (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:642:12)
    at Object.queryData (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:652:12)
From previous event:
    at Object.exports.queryData (/usr/local/bin/lib/engine.js.jx:671:27)
    at /Users/rjbain/kalabox/apps/usfca-public/plugins/kalabox-plugin-pantheon/lib/env.js:402:32
From previous event:
    at /Users/rjbain/kalabox/apps/usfca-public/plugins/kalabox-plugin-pantheon/lib/env.js:396:10
    at /usr/local/bin/lib/core/events.js.jx:149:11
    at /usr/local/bin/lib/core/events.js.jx:148:24
From previous event:
    at /usr/local/bin/lib/core/events.js.jx:132:23
From previous event:
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:129:18)
    at /usr/local/bin/lib/core/events.js.jx:108:19
From previous event:
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:100:21)
From previous event:
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:232:4)
    at /usr/local/bin/lib/app.js.jx:689:24
From previous event:
    at startComponent (/usr/local/bin/lib/app.js.jx:688:4)
    at services.verify.then.then.then.then.map.concurrency (/usr/local/bin/lib/app.js.jx:735:12)
From previous event:
    at Object.exports.start (/usr/local/bin/lib/app.js.jx:734:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/start.js.jx:22:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:876:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:173:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:236:12)
    at process._tickCallback (node.js:812:13)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:235:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:244:1)
    at Module._compile (module.js:518:26)
    at readX (module.js:1284:5)
    at Object.Module._extensions..jx (module.js:1711:10)
    at Module.load (module.js:345:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:377:17)
    at require (module.js:395:17)
    at Object.<anonymous> (_jx_source:17:3)
    at Module._compile (module.js:518:26)
    at Module.load (module.js:353:12)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:733:12)
    at startup (node.js:371:18)
    at node.js:1464:3

````

**Comment by jdleonard** [comment]
I am now also encountering this.

**Comment by reynoldsalec** [comment]
Once he's out of the code dungeon @pirog can confirm this, but I *think* the 0.11.0 release he's working on now may resolve this type of issue.

**Comment by pirog** [comment]
Yeah this should be fixed in v0.11.0 which is coming out on Friday.
